### ISSUE TITLE
[Bug] Broken API

### ISSUE THREAD

**Issue by antshar** [texstudio-org/texstudio#2318]
In new versions you can no longer access 
```
pdfs[0].widget()
```
via scripting. Therefore, cannot use its methods. Getting the following error
![image](https://user-images.githubusercontent.com/54121465/167343343-cf0c886c-0897-439e-9816-4d4225ae88e4.png)

Whereas on TeXstudio 2.12.22 for example you can access this property
![image](https://user-images.githubusercontent.com/54121465/167343424-ebd7f516-6e64-4405-b944-0f45acd91fd8.png)

- TeXstudio: 4.2.3
- Qt: 5
- OS: Windows 10
- TeX distribution: TeXLive

**Comment by sunderme** [comment]
fixed.
However, I have no test code for testing script/pdf, so the code concerning this is basically untested.
please check with the latest dev. build

**Comment by antshar** [comment]
@sunderme where can I find it? Should I build binaries myself to test?

**Comment by muzimuzhi** [comment]
@antshar Artifacts on webpage https://github.com/texstudio-org/texstudio/actions/runs/2295777499, in general https://github.com/texstudio-org/texstudio/actions/workflows/cd.yml .

**Comment by antshar** [comment]
@muzimuzhi thanks!

**Comment by antshar** [comment]
@sunderme yeah, now I can access those properties and methods.
However, during the testing I noticed one difference with the behavior on older versions:
I have the following snippet script that allows me to inspect all undocumented objects
```
%SCRIPT
function write_properties(obj) {
    app.fileNew();
    newEditor = documentManager.currentDocument.editorView.editor;
    newEditor.setText(Object.getOwnPropertyNames(obj).join("\n"));
}
obj = pdfs[0].widget();
write_properties(obj);
```
In older versions I'm having these as an output, i.e it's clearly distinguishable what's a property and what's a function due to parentheses
![image](https://user-images.githubusercontent.com/54121465/167714214-46b01584-71f1-4d99-bf87-fb911a232b26.png)

Whereas now it's
![image](https://user-images.githubusercontent.com/54121465/167714762-6cb4f1f4-35d8-4ef5-a67a-97864248e559.png)
 Also it seems like some of properties\methods are gone. There used to be 174 lines in that newly created by the script tab, now it's 163 lines

Of course, luckily I can access `pdfs[0].widget()` object again, but it still worth mentioning because something really got changed in source code.
